sound/soc/sof/intel/hda-loader-skl.c

```diff
@@ -409,13 +408,6 @@ cl_skl_cldma_copy_to_buf(struct snd_sof_dev *sdev, const void *bin,
 			cl_skl_cldma_fill_buffer(sdev, bufsize, bufsize,
```

Comment:
You can remove one of the two arguments which you test for anyways:
unsigned int size = (bufsize == copysize) ? bufsize : copysize;

Reply:
OK, done.